packages/web-api-client/src/a11y-service-client.ts

```diff
@@ -43,9 +43,29 @@ export class A11yServiceClient {
     }
 
     public async postScanUrl(scanUrl: string, priority?: number): Promise<ResponseWithBodyType<ScanRunResponse[]>> {
-        const requestBody: ScanRunRequest[] = [{ url: scanUrl, priority: priority === undefined ? 0 : priority }];
+        const requestBody: ScanRunRequest = { url: scanUrl };
```

Comment:
could we rename this to something like `scanRequest` since it's not the actual body of the request by itself?

Reply:
changes show in most second most recent commit